apps/worker/src/bootstrap.ts

```diff
@@ -71,6 +95,8 @@ export async function bootstrap(): Promise<INestApplication> {
 
   await app.listen(process.env.PORT);
 
+  await startAppInfra(app);
```

Comment:
Doubting if we should enable the worker before or after letting the application to listen to requests.

Reply:
I think that this order makes sense here